ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.tsx

```diff
@@ -455,48 +473,45 @@ function ClustersTablePanel({
     // After there is a response, if there are clusters or search filter.
     // Conditionally render a subsequent error in addition to most recent successful respnse.
     return (
-        <div className="overflow-hidden w-full">
-            {pageHeader}
-            <PanelNew testid="panel">
-                <PanelHead>
-                    {headerComponent}
-                    <PanelHeadEnd>{headerActions}</PanelHeadEnd>
-                </PanelHead>
-                <PanelBody>
-                    {errorMessage && (
-                        <Alert
-                            variant="warning"
-                            isInline
-                            title="Unable to fetch clusters"
-                            component="p"
-                        >
-                            {errorMessage}
-                        </Alert>
-                    )}
-                    {messages.length > 0 && (
-                        <div className="flex flex-col w-full items-center bg-warning-200 text-warning-8000 justify-center font-700 text-center">
-                            {messages}
-                        </div>
-                    )}
-                    <div data-testid="clusters-table" className="h-full w-full">
-                        <CheckboxTable
-                            ref={(table) => {
-                                setTableRef(table);
-                            }}
-                            rows={currentClusters}
-                            columns={clusterColumns}
-                            onRowClick={setSelectedClusterId}
-                            toggleRow={toggleCluster}
-                            toggleSelectAll={toggleAllClusters}
-                            selection={checkedClusterIds}
-                            selectedRowId={selectedClusterId}
-                            noDataText="No clusters to show."
-                            minRows={20}
-                            pageSize={pageSize}
-                        />
+        <>
+            <PageSection variant="light" component="div">
+                <Title headingLevel="h1">Clusters</Title>
+                {pageHeader}
+                {headerActions}
+            </PageSection>
+            <Divider component="div" />
+            <PageSection variant="light" isFilled>
+                {errorMessage && (
+                    <Alert
+                        variant="warning"
+                        isInline
+                        title="Unable to fetch clusters"
+                        component="p"
+                    >
+                        {errorMessage}
+                    </Alert>
+                )}
+                {messages.length > 0 && (
+                    <div className="flex flex-col w-full items-center bg-warning-200 text-warning-8000 justify-center font-700 text-center">
```

Comment:
1. Do we want to try to replace Tailwind classes with PF utility classes here, too?
2. If we elect to keep TW classes here for now, we should fix `text-warning-8000`, which I think should be `text-warning-800`.

Reply:
Good eyes. Because it is used only for failure to upgrade single cluster, I can replace as a follow up.